Uplift/Controllers/FavoriteGymsController.swift

```diff
@@ -78,7 +78,7 @@ class OnboardingGymsViewController: UIViewController {
         gymsTableView = UITableView(frame: .zero, style: .plain)
         gymsTableView.delegate = self
         gymsTableView.dataSource = self
-        gymsTableView.register(FavoriteGymCell.self, forCellReuseIdentifier: FavoriteGymCell.reuseIdentifier)
+        gymsTableView.register(FavoriteCell.self, forCellReuseIdentifier: FavoriteCell.identifier)
```

Comment:
We should be storing `FavoriteCell`'s identifier in `Constants -> Identifiers` with the other identifiers